pipeline_dp/dataset_histograms/sum_histogram_computation.py

```diff
@@ -247,18 +268,27 @@ def _compute_linf_sum_contributions_histogram(
                             "Drop privacy_id, partition_key")
     # col: (index, float)
 
-    return _compute_frequency_histogram_per_key(
-        col, backend, hist.HistogramType.LINF_SUM_CONTRIBUTIONS,
-        NUMBER_OF_BUCKETS)
+    col = backend.to_multi_transformable_collection(col)
+
+    return backend.flatten((_compute_frequency_histogram_per_key(
```

Comment:
maybe extract and save results of `_compute_frequency_histogram_per_key` into variables to improve readability a little